src/watch_interfaces/CreateBoardControls.py

```diff
@@ -0,0 +1,23 @@
+import RPi.GPIO as GPIO
+
+from src.watch_constants.BoardConstants import *
+
+
+class CreateBoardControls:
+    def __init__(self):
+        GPIO.setmode(GPIO.BCM)
+        GPIO.setup(BUTTONS, GPIO.IN, pull_up_down=GPIO.PUD_UP)
+
+    def handle_action(self, pin):
+        label = LABELS[BUTTONS.index(pin)]
+        if pin == LABELS[BUTTONS.index(pin)]:
+            print("Button press detected on pin: {} label: {}".format(pin, label))
+
+        if label == 'A':
```

Comment:
if chain, can have unwanted consquences if you aren't careful. However the if chain itself is fine just in event based programming you can trigger multiple things at once which can data race